xchange-idex/src/test/java/org/knowm/xchange/idex/IdexExchangeTest.java

```diff
@@ -0,0 +1,34 @@
+package org.knowm.xchange.idex;
+
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.knowm.xchange.Exchange;
+import org.knowm.xchange.ExchangeFactory;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.meta.ExchangeMetaData;
+
+import java.util.List;
+
+public class IdexExchangeTest {
```

Comment:
Test name should be suffixed `..Integration` so it's run with failsafe instead of surefire. Otherwise if IDEX is down this test will fail and break the build.